### PR TITLE
feat(eslint-plugin-gamut)!: gamut eslint-pkg + add no .scss / themed rules

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview

Creates `eslint-plugin-gamut` and adds '`gamut/prefer-themed` &   `gamut/no-css-standalone`  lint rules to enforce better gamut practices, and moves the existing rule `import-path` over into the new package.

Most of this package is autofixed lint rules and that kind of thing - monolith [here](https://github.com/codecademy-engineering/Codecademy/pull/29228)

### PR Checklist

- [x] Related to designs:
- [x] Related to JIRA ticket: web-1939
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

1. Run gamut locally
2.  Run yarn run lint
3. You should see no lint errors (other then the ones we want)

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.31-alpha.90232d.0<br/>@codecademy/gamut-labs@28.4.0-alpha.90232d.0<br/>@codecademy/styleguide@52.2.0-alpha.90232d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61b7bccff2d608239e257ab7--gamut-preview.netlify.app>https://61b7bccff2d608239e257ab7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61b7bccff2d608239e257ab7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61b8ac40b396290df2096474--gamut-preview.netlify.app>https://61b8ac40b396290df2096474--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61b8ac40b396290df2096474>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61ba73fe8b0c0fabb70f2169--gamut-preview.netlify.app>https://61ba73fe8b0c0fabb70f2169--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61ba73fe8b0c0fabb70f2169>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.a7c045.0<br/>@codecademy/gamut-kit@0.6.50-alpha.a7c045.0<br/>@codecademy/gamut-labs@29.2.1-alpha.a7c045.0<br/>@codecademy/gamut@44.1.1-alpha.a7c045.0<br/>@codecademy/markdown-overrides@0.8.1-alpha.a7c045.0<br/>@codecademy/monaco@0.6.3-alpha.a7c045.0<br/>@codecademy/styleguide@54.0.5-alpha.a7c045.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61f423bdb9b09f048c330dbe--gamut-preview.netlify.app>https://61f423bdb9b09f048c330dbe--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61f423bdb9b09f048c330dbe>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.8432ac.0<br/>@codecademy/gamut-kit@0.6.50-alpha.8432ac.0<br/>@codecademy/gamut-labs@29.2.1-alpha.8432ac.0<br/>@codecademy/gamut@44.1.1-alpha.8432ac.0<br/>@codecademy/markdown-overrides@0.8.1-alpha.8432ac.0<br/>@codecademy/monaco@0.6.3-alpha.8432ac.0<br/>@codecademy/styleguide@54.0.5-alpha.8432ac.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.f712aa.0<br/>@codecademy/gamut-kit@0.6.51-alpha.f712aa.0<br/>@codecademy/gamut-labs@29.2.2-alpha.f712aa.0<br/>@codecademy/gamut@44.2.1-alpha.f712aa.0<br/>@codecademy/markdown-overrides@0.8.1-alpha.f712aa.0<br/>@codecademy/monaco@0.6.4-alpha.f712aa.0<br/>@codecademy/styleguide@54.0.6-alpha.f712aa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61f427a549378717ac012cff--gamut-preview.netlify.app>https://61f427a549378717ac012cff--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61f427a549378717ac012cff>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61f427d6d5267f1e034f10c8--gamut-preview.netlify.app>https://61f427d6d5267f1e034f10c8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61f427d6d5267f1e034f10c8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fbec6949920ccae84485e3--gamut-preview.netlify.app>https://61fbec6949920ccae84485e3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fbec6949920ccae84485e3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.db71ea.0<br/>@codecademy/gamut-kit@0.6.56-alpha.db71ea.0<br/>@codecademy/gamut-labs@29.2.7-alpha.db71ea.0<br/>@codecademy/gamut@44.3.2-alpha.db71ea.0<br/>@codecademy/markdown-overrides@0.8.2-alpha.db71ea.0<br/>@codecademy/monaco@0.6.9-alpha.db71ea.0<br/>@codecademy/styleguide@54.1.2-alpha.db71ea.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fd4a9304d7f02f35a8e069--gamut-preview.netlify.app>https://61fd4a9304d7f02f35a8e069--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fd4a9304d7f02f35a8e069>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.09ea88.0<br/>@codecademy/gamut-kit@0.6.57-alpha.09ea88.0<br/>@codecademy/gamut-labs@29.2.8-alpha.09ea88.0<br/>@codecademy/gamut@44.4.0-alpha.09ea88.0<br/>@codecademy/markdown-overrides@0.8.2-alpha.09ea88.0<br/>@codecademy/monaco@0.6.10-alpha.09ea88.0<br/>@codecademy/styleguide@54.2.0-alpha.09ea88.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fd8cb977b14a00765f83f1--gamut-preview.netlify.app>https://61fd8cb977b14a00765f83f1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fd8cb977b14a00765f83f1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.c94951.0<br/>@codecademy/gamut-kit@0.6.58-alpha.c94951.0<br/>@codecademy/gamut-labs@29.2.9-alpha.c94951.0<br/>@codecademy/gamut@44.4.1-alpha.c94951.0<br/>@codecademy/markdown-overrides@0.8.2-alpha.c94951.0<br/>@codecademy/monaco@0.6.11-alpha.c94951.0<br/>@codecademy/styleguide@54.2.1-alpha.c94951.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fd9159ad854a0b1c9a3dfa--gamut-preview.netlify.app>https://61fd9159ad854a0b1c9a3dfa--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fd9159ad854a0b1c9a3dfa>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.b560ba.0<br/>@codecademy/gamut-kit@0.6.59-alpha.b560ba.0<br/>@codecademy/gamut-labs@29.2.10-alpha.b560ba.0<br/>@codecademy/gamut@44.5.1-alpha.b560ba.0<br/>@codecademy/markdown-overrides@0.8.2-alpha.b560ba.0<br/>@codecademy/monaco@0.6.12-alpha.b560ba.0<br/>@codecademy/styleguide@54.2.2-alpha.b560ba.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fd9d8be12dde0cdc3262c2--gamut-preview.netlify.app>https://61fd9d8be12dde0cdc3262c2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fd9d8be12dde0cdc3262c2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.935e74.0<br/>@codecademy/gamut-kit@0.6.59-alpha.935e74.0<br/>@codecademy/gamut-labs@29.2.10-alpha.935e74.0<br/>@codecademy/gamut@44.5.1-alpha.935e74.0<br/>@codecademy/markdown-overrides@0.8.2-alpha.935e74.0<br/>@codecademy/monaco@0.6.12-alpha.935e74.0<br/>@codecademy/styleguide@54.2.2-alpha.935e74.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fda231ccb6a10e612c786a--gamut-preview.netlify.app>https://61fda231ccb6a10e612c786a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fda231ccb6a10e612c786a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fdb6184afac325bb4ead42--gamut-preview.netlify.app>https://61fdb6184afac325bb4ead42--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fdb6184afac325bb4ead42>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fdb838bc4e7f21408d691f--gamut-preview.netlify.app>https://61fdb838bc4e7f21408d691f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fdb838bc4e7f21408d691f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.d2732e.0<br/>@codecademy/gamut-kit@0.6.59-alpha.d2732e.0<br/>@codecademy/gamut-labs@29.2.10-alpha.d2732e.0<br/>@codecademy/gamut@44.5.1-alpha.d2732e.0<br/>@codecademy/markdown-overrides@0.8.2-alpha.d2732e.0<br/>@codecademy/monaco@0.6.12-alpha.d2732e.0<br/>@codecademy/styleguide@54.2.2-alpha.d2732e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61fdb95ea4a91d28b7c30ca9--gamut-preview.netlify.app>https://61fdb95ea4a91d28b7c30ca9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61fdb95ea4a91d28b7c30ca9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.1ab621.0<br/>@codecademy/gamut-kit@0.6.66-alpha.1ab621.0<br/>@codecademy/gamut-labs@29.3.7-alpha.1ab621.0<br/>@codecademy/gamut@44.7.1-alpha.1ab621.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.1ab621.0<br/>@codecademy/monaco@0.6.17-alpha.1ab621.0<br/>@codecademy/styleguide@54.3.7-alpha.1ab621.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://621961a928db393f8404afc0--gamut-preview.netlify.app>https://621961a928db393f8404afc0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/621961a928db393f8404afc0>Deploy Logs</a></p>

File: jest.config.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+    '<rootDir>/packages/gamut-kit',
+    '<rootDir>/packages/gamut-labs',
+    '<rootDir>/packages/gamut-patterns',
+    '<rootDir>/packages/gamut-styles',
+    '<rootDir>/packages/gamut-tests',
+    '<rootDir>/packages/macros',
+    '<rootDir>/packages/markdown-overrides',
+    '<rootDir>/packages/monaco',
+    '<rootDir>/packages/styleguide',
+    '<rootDir>/packages/variance',
```
Comment (jakemhiller) [inline jest.config.js:16]: y?

File: packages/eslint-plugin-gamut/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "scripts": {
+    "verify": "tsc --noEmit",
+    "build:compile": "tsc",
+    "build:clean": "rm -rf dist",
+    "build": "yarn build:clean && yarn build:compile",
+    "lernaBuildTask": "yarn build"
+  },
+  "types": "./dist/index.d.ts",
+  "dependencies": {
+    "@types/eslint": "^8.2.1",
```
Comment (jakemhiller) [inline packages/eslint-plugin-gamut/package.json:31]: if this is an actual dependency, could we remove it from `devDependencies`? (or the other way around, if it's only a dev dependency)

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/index.tsx [hunk 3]
```diff
@@ -8,9 +8,9 @@ import { LegalLinks } from './LegalLinks';
 import { MadeIn } from './MadeIn';
 
 const LegalLocalContainer = styled.div`
-  border-top: 1px solid ${({ theme }) => theme.colors.navy};
+  border-top: 1px solid ${themed('colors.navy')};
   font-size: 0.875rem;
-  padding: 1rem 0 0.5rem;
+  padding: 1rem 0 0.5rem;d
```
[the remark this answers is not in the document]
Comment (dreamwasp) [reply packages/gamut-labs/src/GlobalFooter/FooterLegal/index.tsx:13]: it is a typo 😅

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.02ea57.0<br/>@codecademy/gamut-kit@0.6.71-alpha.02ea57.0<br/>@codecademy/gamut-labs@30.0.3-alpha.02ea57.0<br/>@codecademy/gamut@45.0.2-alpha.02ea57.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.02ea57.0<br/>@codecademy/monaco@0.6.20-alpha.02ea57.0<br/>@codecademy/styleguide@55.0.3-alpha.02ea57.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6227a0030d085f0eb03789f6--gamut-preview.netlify.app>https://6227a0030d085f0eb03789f6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6227a0030d085f0eb03789f6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.3ce5c8.0<br/>@codecademy/gamut-kit@0.6.79-alpha.3ce5c8.0<br/>@codecademy/gamut-labs@30.2.5-alpha.3ce5c8.0<br/>@codecademy/gamut@45.1.4-alpha.3ce5c8.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.3ce5c8.0<br/>@codecademy/monaco@0.6.25-alpha.3ce5c8.0<br/>@codecademy/styleguide@55.1.5-alpha.3ce5c8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6233588423391335c873b54f--gamut-preview.netlify.app>https://6233588423391335c873b54f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6233588423391335c873b54f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6233627aa1e8444853aba2cc--gamut-preview.netlify.app>https://6233627aa1e8444853aba2cc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6233627aa1e8444853aba2cc>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@0.0.2-alpha.986dd0.0<br/>@codecademy/gamut-kit@0.6.81-alpha.986dd0.0<br/>@codecademy/gamut-labs@30.2.7-alpha.986dd0.0<br/>@codecademy/gamut@45.2.2-alpha.986dd0.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.986dd0.0<br/>@codecademy/monaco@0.6.27-alpha.986dd0.0<br/>@codecademy/styleguide@55.1.7-alpha.986dd0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6233844d5df8a12a8206b62c--gamut-preview.netlify.app>https://6233844d5df8a12a8206b62c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6233844d5df8a12a8206b62c>Deploy Logs</a></p>

**Review by jakemhiller — APPROVED** [review]

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623393249164b800c6f47619--gamut-preview.netlify.app>https://623393249164b800c6f47619--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623393249164b800c6f47619>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6233ad842da1c223cf424926--gamut-preview.netlify.app>https://6233ad842da1c223cf424926--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6233ad842da1c223cf424926>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.4d7b12.0<br/>@codecademy/gamut-kit@0.6.82-alpha.4d7b12.0<br/>@codecademy/gamut-labs@30.3.1-alpha.4d7b12.0<br/>@codecademy/gamut@45.2.2-alpha.4d7b12.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.4d7b12.0<br/>@codecademy/monaco@0.6.27-alpha.4d7b12.0<br/>@codecademy/styleguide@55.2.1-alpha.4d7b12.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6234ad8457dd40007ebf08d0--gamut-preview.netlify.app>https://6234ad8457dd40007ebf08d0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6234ad8457dd40007ebf08d0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.634d66.0<br/>@codecademy/gamut-kit@0.6.82-alpha.634d66.0<br/>@codecademy/gamut-labs@30.3.1-alpha.634d66.0<br/>@codecademy/gamut@45.2.2-alpha.634d66.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.634d66.0<br/>@codecademy/monaco@0.6.27-alpha.634d66.0<br/>@codecademy/styleguide@55.2.1-alpha.634d66.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.546df6.0<br/>@codecademy/gamut-kit@0.6.82-alpha.546df6.0<br/>@codecademy/gamut-labs@30.3.1-alpha.546df6.0<br/>@codecademy/gamut@45.2.2-alpha.546df6.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.546df6.0<br/>@codecademy/monaco@0.6.27-alpha.546df6.0<br/>@codecademy/styleguide@55.2.1-alpha.546df6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6234bbb99199900cb65d7049--gamut-preview.netlify.app>https://6234bbb99199900cb65d7049--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6234bbb99199900cb65d7049>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.84d9c7.0<br/>@codecademy/gamut-kit@0.6.82-alpha.84d9c7.0<br/>@codecademy/gamut-labs@30.3.1-alpha.84d9c7.0<br/>@codecademy/gamut@45.2.2-alpha.84d9c7.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.84d9c7.0<br/>@codecademy/monaco@0.6.27-alpha.84d9c7.0<br/>@codecademy/styleguide@55.2.1-alpha.84d9c7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6234da078ff75900639e7fd2--gamut-preview.netlify.app>https://6234da078ff75900639e7fd2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6234da078ff75900639e7fd2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.f96830.0<br/>@codecademy/gamut-kit@0.6.82-alpha.f96830.0<br/>@codecademy/gamut-labs@30.3.1-alpha.f96830.0<br/>@codecademy/gamut@45.2.2-alpha.f96830.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.f96830.0<br/>@codecademy/monaco@0.6.27-alpha.f96830.0<br/>@codecademy/styleguide@55.2.1-alpha.f96830.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623502019d5c3c2fa4ed54fd--gamut-preview.netlify.app>https://623502019d5c3c2fa4ed54fd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623502019d5c3c2fa4ed54fd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.5fde3b.0<br/>@codecademy/gamut-kit@0.6.82-alpha.5fde3b.0<br/>@codecademy/gamut-labs@30.3.1-alpha.5fde3b.0<br/>@codecademy/gamut@45.2.2-alpha.5fde3b.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.5fde3b.0<br/>@codecademy/monaco@0.6.27-alpha.5fde3b.0<br/>@codecademy/styleguide@55.2.1-alpha.5fde3b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6238d827edeac500b6b373fe--gamut-preview.netlify.app>https://6238d827edeac500b6b373fe--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6238d827edeac500b6b373fe>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalFooter/FooterLegal/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MattSchiller) [reply packages/gamut-labs/src/GlobalFooter/FooterLegal/index.tsx]: Or an emoji licking its own nose ;d

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@3.0.3-alpha.11c863.0<br/>@codecademy/gamut-kit@0.6.84-alpha.11c863.0<br/>@codecademy/gamut-labs@30.4.1-alpha.11c863.0<br/>@codecademy/gamut@45.2.2-alpha.11c863.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.11c863.0<br/>@codecademy/monaco@0.6.27-alpha.11c863.0<br/>@codecademy/styleguide@55.2.3-alpha.11c863.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623a1ede7c90f3006d988b84--gamut-preview.netlify.app>https://623a1ede7c90f3006d988b84--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623a1ede7c90f3006d988b84>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@0.0.2-alpha.0ac718.0<br/>@codecademy/gamut-kit@0.6.84-alpha.0ac718.0<br/>@codecademy/gamut-labs@30.4.1-alpha.0ac718.0<br/>@codecademy/gamut@45.2.2-alpha.0ac718.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.0ac718.0<br/>@codecademy/monaco@0.6.27-alpha.0ac718.0<br/>@codecademy/styleguide@55.2.3-alpha.0ac718.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@1.0.1-alpha.62ebd2.0<br/>@codecademy/gamut-kit@0.6.84-alpha.62ebd2.0<br/>@codecademy/gamut-labs@30.4.1-alpha.62ebd2.0<br/>@codecademy/gamut@45.2.2-alpha.62ebd2.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.62ebd2.0<br/>@codecademy/monaco@0.6.27-alpha.62ebd2.0<br/>@codecademy/styleguide@55.2.3-alpha.62ebd2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623a3040ee3a2c1f205e3514--gamut-preview.netlify.app>https://623a3040ee3a2c1f205e3514--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623a3040ee3a2c1f205e3514>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@1.0.1-alpha.d43a6c.0<br/>@codecademy/gamut-kit@0.6.86-alpha.d43a6c.0<br/>@codecademy/gamut-labs@30.5.1-alpha.d43a6c.0<br/>@codecademy/gamut@45.3.1-alpha.d43a6c.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.d43a6c.0<br/>@codecademy/monaco@0.6.28-alpha.d43a6c.0<br/>@codecademy/styleguide@55.4.1-alpha.d43a6c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623c8c0e971a3d00bc4c30ec--gamut-preview.netlify.app>https://623c8c0e971a3d00bc4c30ec--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623c8c0e971a3d00bc4c30ec>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@1.0.1-alpha.d5d6ba.0<br/>@codecademy/gamut-kit@0.6.88-alpha.d5d6ba.0<br/>@codecademy/gamut-labs@30.6.2-alpha.d5d6ba.0<br/>@codecademy/gamut@45.4.1-alpha.d5d6ba.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.d5d6ba.0<br/>@codecademy/monaco@0.6.29-alpha.d5d6ba.0<br/>@codecademy/styleguide@55.4.3-alpha.d5d6ba.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623dcd07418a160fbeb4191f--gamut-preview.netlify.app>https://623dcd07418a160fbeb4191f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623dcd07418a160fbeb4191f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623dcf406f773905def1193d--gamut-preview.netlify.app>https://623dcf406f773905def1193d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623dcf406f773905def1193d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@1.0.1-alpha.e5d3f8.0<br/>@codecademy/gamut-kit@0.6.88-alpha.e5d3f8.0<br/>@codecademy/gamut-labs@30.6.2-alpha.e5d3f8.0<br/>@codecademy/gamut@45.4.1-alpha.e5d3f8.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.e5d3f8.0<br/>@codecademy/monaco@0.6.29-alpha.e5d3f8.0<br/>@codecademy/styleguide@55.4.3-alpha.e5d3f8.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@1.0.1-alpha.f8e39d.0<br/>@codecademy/gamut-kit@0.6.88-alpha.f8e39d.0<br/>@codecademy/gamut-labs@30.6.2-alpha.f8e39d.0<br/>@codecademy/gamut@45.4.1-alpha.f8e39d.0<br/>@codecademy/markdown-overrides@0.8.3-alpha.f8e39d.0<br/>@codecademy/monaco@0.6.29-alpha.f8e39d.0<br/>@codecademy/styleguide@55.4.3-alpha.f8e39d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://623dd953161d27206470d924--gamut-preview.netlify.app>https://623dd953161d27206470d924--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/623dd953161d27206470d924>Deploy Logs</a></p>

**Pull request merged**